lib/install/validate/network.go

```diff
@@ -533,9 +535,13 @@ func (v *Validator) inDVP(op trace.Operation, host types.ManagedObjectReference,
 	return false
 }
 
-// checkVDSMembership verifes all hosts in the vCenter are connected to the vDS
-func (v *Validator) checkVDSMembership(op trace.Operation, network types.ManagedObjectReference, netName string) error {
-	defer trace.End(trace.Begin(network.Value, op))
+// checkVDSMembership verifies all hosts in the vCenter are connected to the vDS
+func (v *Validator) checkVDSMembership(op trace.Operation, network object.NetworkReference, netName string) error {
+	defer trace.End(trace.Begin(network.Reference().Value, op))
+
+	if _, dpg := network.(object.DistributedVirtualPortgroup); !dpg {
+		return nil
```

Comment:
Is there no way to check host connectivity for an OpaqueNetwork? At some point we will want to do so in the `pkg/vsphere/compute/placement:CheckHost` logic.

Reply:
I think I can try, I checked there is host property in the OpaqueNetwork object, so we can add the connectivity validation.